src/main/java/com/ericsson/ei/controller/AggregatedObjectControllerImpl.java

```diff
@@ -77,4 +87,38 @@
             return new ResponseEntity<>(errorJsonAsString, HttpStatus.INTERNAL_SERVER_ERROR);
         }
     }
+
+    @Override
+    @CrossOrigin
+    @ApiOperation(value = "Perform a freestyle query to retrieve aggregated objects")
+    public ResponseEntity<?> createAggregatedObjectsQuery(@RequestBody final QueryBody body,
```

Comment:
You actually do not create the query at the same time it does.
Maybe this should be named performQueryForAgregatedObjects

Reply:
I am not sure that raml can be made to throw out a specific name...?

Reply:
This method won't be affected by RAML though? It's only the Controller classes which RAML plugin generates automatically.

Reply:
But the controller's method name is also generated by the RAML...